apps/elasticsearch.go

```diff
@@ -271,15 +268,17 @@ func (p LoggingProcessorElasticsearchGC) Components(ctx context.Context, tag, ui
 	return c
 }
 
-type LoggingReceiverElasticsearchJson struct {
-	LoggingProcessorElasticsearchJson `yaml:",inline"`
-	ReceiverMixin                     confgenerator.LoggingReceiverFilesMixin `yaml:",inline"`
+type LoggingReceiverMacroElasticsearchJson struct {
+	LoggingProcessorMacroElasticsearchJson  `yaml:",inline"`
+	confgenerator.LoggingReceiverFilesMixin `yaml:",inline"`
+	IncludePaths                            []string                      `yaml:"include_paths,omitempty"`
+	MultilineRules                          []confgenerator.MultilineRule `yaml:"multiline_rules,omitempty"`
```

Comment:
`IncludePaths` and `MultilineRules` are properties on the `ReceiverMixin` field of this struct, so we can still assign them as before (e.g. using `r.ReceiverMixin.IncludePaths` and `r.ReceiverMixin.MultilineRules`).

Note: In this specific case the refactor doesn't simplify the code as much since we aren't using `RegisterLoggingFilesProcessorMacro`.